src/main/java/org/jboss/pnc/logprocessor/eventduration/domain/LogEvent.java

```diff
@@ -71,7 +71,10 @@ public String getIdentifier() {
             String processContext = mdc.get(PROCESS_CONTEXT_KEY);
             String eventName = mdc.get(EVENT_NAME_KEY);
             if (processContext == null || processContext.equals("")) {
-                warnCounter.increment();
+                // it can be null in unit tests
```

Comment:
Yup! Let's see.

Reply:
Actually it will always be null since LogEvent is just a regular object and we shouldn't be able to inject anything to it (the MeterRegistry). I removed the counters in LogEvent as a result.